imports/ui/components/sidebar_menu/GradCheckerContainer.js

```diff
@@ -38,11 +38,21 @@ export default GradCheckerContainer = createContainer((props) => {
   // foundation = new Criteria(foundationReq.name, modulesCompleted, foundationReq.requiredMC , foundationReq.isStrictMC, foundationReq.subreq);
   // criteria = new Criteria(foundationReq, modulesCompleted);
 
-  criteria = new Criteria(AY1617,modulesCompleted);
+  // TODO: Alternative implementation of GradChecker - Experiment - @walrys Wenhan
```

Comment:
Note that this is verified by @bsweeyee and @Satrio1610 as well (we edited this together). Commented our experiment to use the current method first.